src/components/ProfileSidebar/ProfileSidebar.jsx

```diff
@@ -0,0 +1,97 @@
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import navigationStyles from '../Navigation/Navigation.module.scss';
+import CoursesIcon from '../../images/ui/NavigationCourses.svg';
+import UsersIcon from '../../images/ui/NavigationUsers.svg';
+import ProfessionsIcon from '../../images/ui/NavigationProfessions.svg';
+import Avatar from '../../images/stubs/Avatar.png';
+
+export default function ProfileSidebar({ isAdmin }) {
+  return (
+    <nav className={navigationStyles.navigation}>
+      <NavLink
+        to="/profile"
+        className={({ isActive }) => {
+          const linkClasses = [navigationStyles.profileItem];
+          if (isActive) linkClasses.push(navigationStyles.profileItem_active);
+          return linkClasses.join(' ');
+        }}
```

Comment:
```suggestion
        className={navigationStyles.profileItem + ' ' + (isActive ? navigationStyles.profileItem_active : '') }  
```
а еще как раз у вас  есть библиотека clsx или classnames как раз для такого